CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Unreleased
 
+## Fixes
```

Comment:
Would this be more of a new feature? With a heading of "The task list pattern is now clearer about incomplete tasks"? Not sure - happy either way.